hack/update-generated-defaulters.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+os::build::setup_env
+
+os::util::ensure::built_binary_exists 'gendefaults'
+
+PREFIX=github.com/openshift/origin
+
+INPUT_DIRS=$(
+  find . -not  \( \( -wholename '*/vendor/*' \) -prune \) -name '*.go' | \
+	xargs grep --color=never -l '+k8s:defaulter-gen=' | \
+	xargs -n1 dirname | \
+	sed "s,^\.,${PREFIX}," | \
+	sort -u | \
+	paste -sd,
+)
+
+EXTRA_PEER_DIRS=$(
+  grep -rl SetDefaults_ vendor/k8s.io/kubernetes/pkg | \
+  xargs dirname | \
+  sort -u | \
+  sed 's,^vendor/,,' | \
+  paste -sd,
+)
+
+gendefaults \
```

Comment:
We need to set `go-header-file` pointing to an empty file, so we don't get defaulters generated with k8s header, like [here](https://github.com/openshift/origin/pull/12143/commits/44e2f6fe9e423285513ee6d67dadf5d8367adcbd)